client/common/cmdline.c

```diff
@@ -3082,6 +3082,17 @@ int freerdp_client_settings_parse_command_line_arguments(rdpSettings* settings,
 		{
 			settings->BitmapCacheEnabled = enable;
 		}
+		CommandLineSwitchCase(arg, "persist-cache")
+		{
+			settings->BitmapCachePersistEnabled = enable;
+		}
+		CommandLineSwitchCase(arg, "persist-cache-file")
```

Comment:
For the file we might set a default in application data dir? We have `GetKnownPath` so that should be good enough to get the proper base directory on all supporte os?

Reply:
That would also require creating a directory to make sure the file can be created inside of it, and which directory should we use by default? Right now you have to explicitly set the bitmap cache file path, but this is an improvement that could come later

Reply:
Most likely a subdirectory of `GetKnownPath(KNOWN_PATH_XDG_CACHE_HOME)`, but true, can be done later on.